src/authorizations/utils/permissions.ts

```diff
@@ -94,6 +94,8 @@ export const allAccessPermissions = (
   return allPermissionTypes.flatMap(withOrgID)
 }
 
+// add a permission string ('read' or 'write' is the second argument)
```

Comment:
(nit): should this comment be here?

Reply:
I agree with @Palakp41 that we do not need comments in here. But i will defer to @jrenee42 on this because she said it made it easier for her to understand.